packages/models-library/src/models_library/api_schemas_resource_usage_tracker/service_runs.py

```diff
@@ -38,3 +38,13 @@ class ServiceRunGet(BaseModel):
 class ServiceRunPage(NamedTuple):
     items: list[ServiceRunGet]
     total: PositiveInt
+
+
+class OsparcCreditsAggregatedByServiceGet(BaseModel):
+    osparc_credits: Decimal
+    service_key: ServiceKey
```

Comment:
version is not foreseen correct?

Reply:
For now no, the service key is the first experiment, and let's see what users say.